src/govuk/components/character-count/character-count.yaml

```diff
@@ -37,6 +37,10 @@ params:
   required: false
   description: Options for the hint component.
   isComponent: true
+- name: fallbackHintText
+  type: string
+  required: false
+  description: Text describing the maximum number of characters or words that can be entered, which is announced to screen readers and displayed as a fallback if the character count JavaScript does not run. Instances of `%{count}` within the hint will be replaced by the value of `maxwords` if configured, otherwise it uses `maxlength`. By default, fallback text is provided in English based on what parameters are provided to the component.
```

Comment:
Might be worth running this past @claireashworth when she's back - it's quite long!

Reply:
I've made the assumption that all this info is needed so it's still long, but the sentences are broken up a bit more. The suggestion is below, but I want to check:
- whether this is specifically a limit on the number of characters that is read out by a screen reader, rather than a general limit that is also read out by screen readers. 
- also, when saying 'if configured', what is the context? Is it something like - 'Depending on how you configure the component and what parameters you add...' ?
Suggested tweaks:
Text describing the maximum number of characters you can enter which will be announced to screen readers. The text is displayed as a fallback if the character count of JavaScript does not run.
Depending on how you configure the component and what parameters you add, instances of `%{count}` are replaced by the value of `maxwords`. If not configured it uses `maxlength`. By default, fallback text is in English.

Reply:
@claireashworth 
1. This is a limit on how much can be entered into the textarea. The description of the limit is announced by screen readers, or is displayed visually as a fallback if JavaScript is unavailable. 
2. Pretty much. The counter uses characters and `maxlength` by default, and will only switch to words mode if the `maxwords` parameter has been explicitly set.

Reply:
```suggestion
  description: Text describing the maximum number of characters you can enter, which is announced to screen readers. The text is displayed as a fallback if the character count JavaScript does not run. Depending on how you configure this component and what parameters you add, instances of `%{count}` are replaced by the value of `maxwords`. If not configured, it uses `maxlength`. By default, fallback text is provided in English.
```